README.md

```diff
@@ -111,11 +111,14 @@ Flags:
       --insecure-skip-tls-verify       If true, the server's certificate will not be checked for validity. This will make your HTTPS connections insecure
       --kubeconfig string              Path to the kubeconfig file to use for CLI requests.
   -n, --namespace string               If present, the namespace scope for this CLI request
+      --node-columns string            Comma-separated list of node columns to display (e.g. 'NAME,CPU,MEM')
+      --pod-columns string             Comma-separated list of pod columns to display (e.g. 'NAMESPACE,POD,STATUS')
       --request-timeout string         The length of time to wait before giving up on a single server request. Non-zero values should contain a corresponding time unit (e.g. 1s, 2m, 3h). A value of zero means don't timeout requests. (default "0")
   -s, --server string                  The address and port of the Kubernetes API server
+      --show-all-columns               If true, show all columns (default true)
```

Comment:
I think this should always be implied (to show all columns) unless the column names are provided with `node-columns` and `pod-columns`.